README.md

```diff
@@ -5095,6 +5116,7 @@ Callback for the periodic and non periodic vehicle data read function.
 |`emergencyEvent`|EmergencyEvent|False|Information related to an emergency event (and if it occurred)|
 |`clusterModeStatus`|ClusterModeStatus|False|The status modes of the cluster|
 |`myKey`|MyKey|False|Information related to the MyKey feature|
+|`windowStatus`|WindowStatus|False|Information related to window status|
```

Comment:
I think the description should indicate that this is an array of `WindowStatus` structs